drjt/My-List-Util/lib/My/List/Util.pm

```diff
@@ -0,0 +1,131 @@
+package My::List::Util;
+
+use 5.010;
+use strict;
+use warnings;
+
+use List::Util qw( sum shuffle );
```

Comment:
The book does suggest looking at `List::Util` for an implementation. But yeah, originally I was planning on just using `reduce` for the sum, but figured that these functions were better suited for the job.